src/Api/Model/Shared/Mapper/MongoQueries.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Api\Model\Shared\Mapper;
+
+class MongoQueries
+{
+    public static function countEntries($databaseName, $collectionName)
+    {
+        $db = MongoStore::connect($databaseName);
```

Comment:
not sure what the cost is, but should we be connecting to the db in each of these functions? would it be better to connect once in the encode function and pass the db into each count method?

Reply:
Good idea. Done.